plugins.go

```diff
@@ -1,37 +0,0 @@
-package main
-
-import (
-	"log"
-	"path/filepath"
-
-	"github.com/kardianos/osext"
-)
-
-// globalPluginDirs returns directories that should be searched for
-// globally-installed plugins (not specific to the current configuration).
-//
-// Earlier entries in this slice get priority over later when multiple copies
-// of the same plugin version are found, but newer versions always override
-// older versions where both satisfy the provider version constraints.
-func globalPluginDirs() []string {
```

Comment:
The motivation for this was to continue to support the _old_ search paths for a while after we release 0.10, in addition to the new ones. But if we're comfortable with just documenting this as a breaking change then that's fine too... just wanted to make it explicit that this is a breaking change.